docker/compose/mozdef_meteor/Dockerfile

```diff
@@ -2,9 +2,10 @@ FROM centos:7
 
 MAINTAINER mozdef@mozilla.com
 
+#updated meteor version to avoid EXDEV: cross-device link not permitted errors
 ENV NODE_VERSION 4.7.0
-ENV METEOR_VERSION 1.4.0.1
-ENV METEOR_FILE_VERSION 1.4.0-1
+ENV METEOR_VERSION 1.4.4.2
```

Comment:
Do we need to make these changes to https://github.com/mozilla/MozDef/blob/master/docker/Dockerfile as well? Since currently, we're supporting both a single container as well as multiple?